setup.py

```diff
@@ -58,13 +58,15 @@ def package_files(directory, strip_leading):
               'Adafruit_PCA9685',
               'adafruit-circuitpython-lis3dh',
               'adafruit-circuitpython-ssd1306',
+              'adafruit-circuitpython-rplidar',
               'RPi.GPIO',
               'pyserial',
           ],
           'nano': [
               'Adafruit_PCA9685',
               'adafruit-circuitpython-lis3dh',
               'adafruit-circuitpython-ssd1306',
+              'adafruit-circuitpython-rplidar',
```

Comment:
We would need corresponding changes also in the conda yaml files (not sure if these packages are available in conda channels or pypi only).

Reply:
Do we need those for conda?  This is only installed on rpi and nano, although I guess we could install it on the host computers as well.   If we only install this on RPi and Nano do we need to update the conda files?